misc/pytorch_toolkit/chest_xray_screening/chest_xray_screening/utils/download_weights.py

```diff
@@ -1,17 +1,47 @@
-from google_drive_downloader import GoogleDriveDownloader as gdd
 from .get_config import get_config
+import os
+import zipfile
+import pycurl
 
 def download_checkpoint():
-    config = get_config(action = 'gdrive')
-    gdd.download_file_from_google_drive(file_id=config['densenet']['file_id'],
-                                    dest_path=config['densenet']['dest_path'],
-                                    unzip=False)
-    gdd.download_file_from_google_drive(file_id=config['densenet_eff']['file_id'],
-                                    dest_path=config['densenet_eff']['dest_path'],
-                                    unzip=False)
+    config = get_config(action = 'download')
+    if not os.path.exists('model_weights'):
+        os.makedirs('model_weights')
+    densenet_url = config['densenet']['url']
+    densenet_path = config['densenet']['dest_path']
+    denseneteff_url = config['densenet_eff']['url']
+    denseneteff_path = config['densenet_eff']['dest_path']
+
+    with open(densenet_path, 'wb') as f:
+        cl = pycurl.Curl()
+        cl.setopt(cl.URL, densenet_url)
+        cl.setopt(cl.WRITEDATA, f)
+        cl.perform()
+        cl.close()
+    with zipfile.ZipFile(config['densenet']['dest_path'], 'r') as zip_ref:
+        zip_ref.extractall('model_weights/')
```

Comment:
For here and on, this action could be extracted in the separate function that downloads file from a given url to a given path and then extracts it. Something like `download_and_extract(url, path)`.